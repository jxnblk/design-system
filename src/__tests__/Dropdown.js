import React from 'react'
import renderer from 'react-test-renderer'
import { Dropdown } from '..'

describe('Dropdown', () => {
  test('renders', () => {
    const json = renderer.create(<Dropdown>Dropdown</Dropdown>).toJSON()
    expect(json).toMatchSnapshot()
  })

  test('renders with "flexDirection" prop', () => {
    const json = renderer
      .create(<Dropdown flexDirection="column">Flex Column Dropdown</Dropdown>)
      .toJSON()
    expect(json).toMatchSnapshot()
    expect(json).toHaveStyleRule('flex-direction', 'column')
  })
})
