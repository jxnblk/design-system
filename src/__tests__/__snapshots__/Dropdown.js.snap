// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown renders 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="c0 "
>
  Dropdown
</div>
`;

exports[`Dropdown renders with "flexDirection" prop 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  className="c0 "
>
  Flex Column Dropdown
</div>
`;
